app/helpers/nav_helper.rb

```diff
@@ -0,0 +1,5 @@
+module NavHelper
+  def nav_menu_collapsed?
+    cookies[:collapsed_nav] == "true"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.